java/java.sourceui/src/org/netbeans/api/java/source/ui/ElementJavadoc.java

```diff
@@ -1280,6 +1282,24 @@ private StringBuilder inlineTags(List<? extends DocTree> tags, TreePath docPath,
                 case TEXT:
                     TextTree ttag = (TextTree)tag;
                     sb.append(ttag.getBody());
+                    break;
+                default : {
+                    // process tags that we cannot add directly because they are not accessible during compilation
+                    switch(tag.getKind().name()) {
+                        case "SUMMARY" : 
+                            try {
+                                Method getSummaryMethod = tag.getClass().getDeclaredMethod("getSummary");
+                                List<? extends DocTree> summaryList = (List<? extends DocTree>)getSummaryMethod.invoke(tag);
+                                sb.append(inlineTags(summaryList, docPath, doc, trees, null));
+                            } catch(NoSuchMethodException | SecurityException | IllegalAccessException | IllegalArgumentException | InvocationTargetException ex) {
```

Comment:
could be: `(ReflectiveOperationException | ... rest)`

`ReflectiveOperationException` is a common supertype which makes those lines shorter.